mkdocs.yml

```diff
@@ -17,6 +17,7 @@ markdown_extensions:
  - admonition
  - attr_list
  - def_list
+ - footnotes
```

Comment:
do these render at the bottom of the entire page?   or within each section?

Reply:
They render at the bottom of the page regardless of where they are defined.

Reply:
okay - that could be a long way away...

Reply:
They are bidirectional if that helps.

I don't mind removing them either. However, they seem like a useful way to include miscellaneous info that may keep readers from getting stuck.

Reply:
certainly reasonable - happy to see how it feels.